speed_games/results_tool/scripts/index.js

```diff
@@ -0,0 +1,293 @@
+    document.querySelector('#input-scene').style.display = 'none'
+    document.querySelector('#waiting-scene').style.display = 'none'
+    document.querySelector('#player-scene').style.display = 'block'
+    displayAndSetPlayers();
+}
+
+
+const setup = () => {
+    document.querySelector('#counter-scene').style.display = 'none'
+    document.querySelector('#scene').style.display = 'none'
+    document.querySelector('#waiting-scene').style.display = 'block'
+    document.querySelector('#player-scene').style.display = 'none'
+}
+
+window.addEventListener('keyup', (event) => {
+    const isConfettitDebugKey = event.key === 'c';
+    const isSpacebarKey = event.code === 'Space'
+    const isSceneSelectionKey = event.code !== 'Space' && Number(event.key) <= CONFIG.SCENES.length
+
+    if (isSceneSelectionKey) {
+        globalVariables.selectedSceneIndex = Number(event.key) - 1
+        const scene = CONFIG.SCENES[Number(event.key) - 1]
+
+        document.querySelector('#player-scene').style.display = 'none'
+        document.querySelector('#counter-scene').style.display = 'none'
+        document.querySelector('#scene').style.display = 'none'
+        document.querySelector('#waiting-scene').style.display = 'block'
+        document.querySelector('#leaderboard-container').innerHTML = ""
+
+        document.querySelector('body').style.backgroundImage = `url(${scene.bg})`;
+        document.querySelector('#waiting-scene').querySelector('.title').innerHTML = scene.title
+        document.querySelector('#scene').querySelector('.background-title').innerHTML = scene.title
+
+    }
+
+    if (isConfettitDebugKey) {
+        shootConfetti()
+    }
+
+
+    if (isSpacebarKey) {
+        const sceneConfig = CONFIG.SCENES[globalVariables.selectedSceneIndex]
+        const candidates = getCandidates();
+        runCountdown();
+        setTimeout(() => {
+            runScene({ title: scene.title, callback: () => { } }, candidates);
+        }, 5000);
+    }
+})
+
+const getCandidates = () => {
+    const scene = CONFIG.SCENES[globalVariables.selectedSceneIndex]
+    let candidates = {}
+    Object.keys(globalVariables.globalInputData).forEach(rowKey => {
+        let values = globalVariables.globalInputData[rowKey][scene.dataKey] ?? [];
+        values = values.filter(x => !Number.isNaN(x) && Number(x) > 0)
+        let medianValue = values.length > 0 ? mean(values) : undefined;
+        if (medianValue !== undefined) {
+            candidates[rowKey] = medianValue
+        } else {
+            console.error(rowKey, ' has no median value!')
+        }
+    })
+    let arr = Object.keys(candidates).map((key, index) => {
+        const playerObject = globalVariables.players.find(p => p.url === key)
+        return {
+            index: playerObject.index,
+            url: key,
+            name: playerObject.name,
+            value: candidates[key],
+            color: playerObject.color
+        }
+    })
+    arr = sortBy(x => x.index, arr)
```

Comment:
This sorts by index, which is a string.

If we have 10 participants the order will be:
1,10,2,3,4,5,6,7,8,9

But I would expect 1,2,3,4,5,6,7,8,9,10